browser/src/App.js

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
-import { NavigationMenu } from './view/components/Navbar/NavigationMenu';
+import { NavigationMenu } from './view/components/NavigationMenu/NavigationMenu';
 
 const navLinks = [
   {
-    name: 'Logo',
+    label: 'Logo',
```

Comment:
We should probably centralize our user-facing strings similar to what we did with GiveShop.

Reply:
Sounds good. I can tackle that in the future in a different story.